Tests/ttLib/woff2_test.py

```diff
@@ -1220,6 +1221,20 @@ def test_roundtrip_no_glyf_and_loca_tables(self):
 		assert tmp.getvalue() == tmp2.getvalue()
 		assert ttFont.flavor == "woff2"
 
+	def test_roundtrip_overlap_bit(self):
```

Comment:
we don't actually "roundtrip" the overlap bit, but we drop it. 
```suggestion
	def test_roundtrip_off_curve_despite_overlap_bit(self):
```